src/melee/gm/gm_17C0.h

```diff
@@ -6,6 +6,7 @@
 #include <melee/gm/forward.h>
 #include <melee/it/forward.h>
 #include <sysdolphin/baselib/forward.h>
+#include "gm/gmtou.static.h"
```

Comment:
Ah, this is the issue. .static.h headers should only be included from .c files because they contain static definitions - any files that include this header have these new static definitions, breaking their data/bss sections